circle.yml

```diff
@@ -85,6 +85,10 @@ jobs:
               --locales en \
               --orientations portrait \
               --timeout 5m
+      - run:
```

Comment:
Not sure if this is necessary. How are we going to commit the changes (when licenses have changed) as part of the CI build? Maybe for now, we can make sure that we run this "manually" as part of the PR checklist. Ultimately, this is something that doesn't change that often. What do you think?

Reply:
Actually, what we could do is something similar as Maps SDK does. Create a `nitpick` step that will 👀 if there are differences (remote vs generate) and it'll make the PR to fail so PR submitter would need to generate (run `make license` locally) and commit the licenses in order for it to pass.

👀 https://github.com/mapbox/mapbox-gl-native/blob/048a07e3c916cfd66b93bd9624e04e437f79649e/platform/android/gradle/android-nitpick.gradle#L13 https://github.com/mapbox/mapbox-gl-native/blob/048a07e3c916cfd66b93bd9624e04e437f79649e/platform/android/gradle/android-nitpick.gradle#L23 which runs https://github.com/mapbox/mapbox-gl-native/blob/048a07e3c916cfd66b93bd9624e04e437f79649e/platform/android/gradle/android-nitpick.gradle#L64 https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/scripts/validate-license.py

Reply:
Yes, I did look into nitpick. I thought it would be good to have it at one point. But, I agree that we can make sure that devs run the license task manually before merging.

Reply:
Looking at Maps approach, it should be pretty straightforward to copy it and we don't need to worry about having to do it manually. It's only adding a new `make nitpick` program and include it as a step for both `build` and `release` jobs.